src/datasets/non_desktop/day_2_7_activation/reference.md

```diff
@@ -0,0 +1,48 @@
+# Non Desktop Day 2 - 7 Activation  
+
+<!-- toc -->
+
+# Introduction
+`firefox_nondesktop_day_2_7_activation` is designed for use in calculating the day 2-7 activation metric, a key result in 2020 for the nondesktop products.
+
+
+# Contents
+`firefox_nondesktop_day_2_7_activation` is an aggregate table with two key metrics: `new_profiles` and `day_2_7_activated`. These are primarily aggregated by `submission_date`.
```

Comment:
We should probably mention the table this is derived from: `nondesktop_clients_last_seen` (we don't have a reference to the non-desktop clients last seen dataset, so let's just reference @jklukas's cookbook which at least mentions it)

```suggestion
`firefox_nondesktop_day_2_7_activation` is a table with two key metrics: `new_profiles` and `day_2_7_activated`, aggregated over `submission_date`. It is derived from the [non-desktop clients last seen table](https://docs.telemetry.mozilla.org/cookbooks/clients_last_seen_bits.html).
```